.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+*Issue #, if available:*
+
+*Description of changes:*
+
+
+By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
```

Comment:
This section I don't like it feel like the contributor will provide his work and we (diagrams-web) can copy and redistribute it with a commercial license... 

Better to remove it, as the source code use a MIT license it should imply that the contribution will be under this license.

Reply:
Your call I am fine with it

Reply:
do you like me to do some of these changes or will you take care?

Reply:
Go ahead, please.